### ISSUE TITLE
Switch arguments to diff-tool

### ISSUE THREAD

**Issue by roesslerj** [approvals/ApprovalTests.Java#26]
Currently, the diff-tool is called with arguments: _.received._ _.approved._, causing the expected result to be on the right side of the merge/diff-tool. Default is that the expected result is to the left. 

Could the order of the arguments be switched, so the approved result shows up on the left?

Great! Thanks!

**Comment by isidore** [comment]
No. Many of the diff tools will not let you move changes from right to
left. Since the approved file is often created in the diff tool by moving
the results you got to the approved side reversing the order is problematic.

Llewellyn Falco
www.approvaltests.com

**Comment by roesslerj** [comment]
Thanks for the fast feedback!
